interface/billing/billing_report.php

```diff
@@ -705,14 +733,14 @@ class='link_submit'  ><?php echo '['.xlt('End Of Day Report').']' ?></a>
     $mmo_num_charges = 0;
 
     foreach ($ret as $iter) {
-        // We include encounters here that have never been billed.  However
+        // We include encounters here that have never been billed. However
         // if it had no selected billing items but does have non-selected
         // billing items, then it is not of interest.
-        if (!$iter['id']) {
-            $res = sqlQuery("SELECT count(*) AS count FROM billing WHERE " .
-            "encounter = ? AND " .
-            "pid=? AND " .
-            "activity = 1", array($iter['enc_encounter'],$iter['enc_pid']));
+        if (! $iter['id']) {
+            $res = sqlQuery("SELECT count(*) AS count FROM billing WHERE " . "encounter = ? AND " . "pid=? AND " . "activity = 1", array(
```

Comment:
I honestly don't know Brady. I know I did it but every since I upgraded my eclipse to ensure I have latest PSR stuff, I get weird stuff. Please don't tell on me to the other developers.:)

I'll try to sort it out.